### ISSUE TITLE
Fix golint issues maya/types/profiles/v1/doc.go

### ISSUE THREAD

**Issue by ashishranjan738** [openebs/openebs#2145]
File: [maya/types/profiles/v1/doc.go](https://github.com/openebs/maya/blob/master/types/profiles/v1/doc.go)

```
Line 17: warning: package comment should be of the form "Package v1 ..." (golint)
```
Resources:
-> Contributors guide: https://github.com/openebs/openebs/blob/master/CONTRIBUTING.md

Complexity: Easy

**Comment by diehard-coder** [comment]
I am working on this issue

**Comment by ashishranjan738** [comment]
sure @DhairyaPatel7  go ahead. let me know if you need any help on [slack](http://slack.openebs.io).

**Comment by ABHIYANSHU** [comment]
Fixed linting issue at #704
